src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -910,8 +910,16 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 			if (!result.matches(editor)) {
 				const targetGroups = editor.hasCapability(EditorInputCapabilities.Untitled) ? this.editorGroupService.groups.map(group => group.id) /* untitled replaces across all groups */ : [groupId];
 				for (const targetGroup of targetGroups) {
+					const resolvedEditor = await this.editorResolverService.resolveEditor({ resource: result.resource, }, targetGroup);
```

Comment:
@lramos15 I think this solution is a bit of a bandaid: ideally `editor.saveAs` and `editor.save` would return an untyped editor input and then `replaceEditors` should not be called with a typed editor but with an untyped one. This is somewhat a missing adoption of untyped editors in general (I was aware of it but forgot about it).

Reply:
I agree. I was just a bit nervous to go with that level refactor given I'm not really sure how sequential vs non sequential save work and all that plays a role. This change while bandaid seemed safe and allows the testing of this flow.

Reply:
I can see in debt week whether a cleaner change is possible.